projects/spectator/schematics/src/spectator/index.ts

```diff
@@ -21,7 +21,7 @@ import { ComponentOptions, DirectiveOptions, ServiceOptions } from './schema';
 export function spectatorComponentSchematic(options: ComponentOptions): Rule {
   return chain([
     externalSchematic('@schematics/angular', 'component', {
-      ...options,
+      ..._omit(options, ['jest', 'withHost', 'withCustomHost']),
```

Comment:
Can you explain, please?

Reply:
Angular 12 complains about options that are passed but not known